### PR TITLE
Significantly nerf Deathnettles

### PR DESCRIPTION
**Pull request by EdenTheLiznerd**
## About the PR

Deathnettles have had their ability to pierce armor be stripped and their damage values brought down alongside how much acid they can hold and how much it injects per hit

## Why / Balance

Deathnettles have proved problematic for any who have had the unfortunate experience of facing them, botanists shouldn't have the power to kill nukies currently, not until they have the proper ways of getting rid of caustic damage and healing, there's also the issue of people trying to powergame nettles constantly. Botanists shouldn't just take down juggernauts and other antags like it's no issue, we have guns, we should be using them (A reminder I am aware of the desire to have nettles require gloves, this is a temporary patch until someone can code a proper system for that)

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
 :cl:
- tweak: Deathnettles are no longer able to bypass armor and don't do as much damage

### CODE REVIEW

**Comment by Aexxie** [comment]
Aexxie approves of this message

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -174,22 +174,21 @@
   - type: MeleeWeapon
     damage:
       types:
-        Heat: 7.5
-        Caustic: 7.5
+        Heat: 4.5
+        Caustic: 4.5
```
[the remark this answers is not in the document]
Comment (EdenTheLiznerd) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:178]: Nettles do purely heat, this is still doing caustic damage was is still harder to heal in a combat situation

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 2]
```diff
@@ -174,22 +174,21 @@
...
-          Quantity: 15
+          Quantity: 9
         - ReagentId: FluorosulfuricAcid
-          Quantity: 15
+          Quantity: 9
   - type: Produce
     seedId: deathNettle
   - type: MeleeChemicalInjector
-    transferAmount: 6
+    transferAmount: 2
```
[the remark this answers is not in the document]
Comment (EdenTheLiznerd) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:190]: If someone wishes to code it they may, I made this PR as a temporary layover until deathnettles recieve an actual rework that makes them require certain items or stops them from just killing fully armored targets with ease

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aexxie) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:178]: I'm going to assume that if you're able to heal in combat, you have advanced stuff. Meshes heal caustic. Crick is right, it should be at least a bit better than normal nettle. Alternatively, you can nerf the nettle too and piss more people off.

**Comment by deltanedas** [comment]
this just makes them worse than nettles in every way

**Comment by Jezithyr** [comment]
I honestly don't think this is necessary with the current state of melee balance. Nerfing armor penetration is a good change but the other nerfs are going a bit too far in the opposite direction. In the current state of combat melee weapons are not very effective even if the melee weapon in question has high kill potential (You won't get close enough to someone with a gun to actually hit them).

Additionally, botany being able to produce weaponizable plants adds alot of potential for in-round chaos and general silliness. Nerfing deathnettles to the point where they are basically unusable in combat is not the solution and seems like a bit of a knee-jerk nerf in response to a certain youtube video ;)

**Comment by asperger-sind** [comment]
> In the current state of combat melee weapons are not very effective even if the melee weapon in question has high kill potential (You won't get close enough to someone with a gun to actually hit them).

Not exactly correct, melee is very strong but yet overshadowed by how strong guns are at close range; you'll be able to get close to a guy with a pistol, get a few swings in, maybe get them to half HP or less if you're using a top-notch melee weapon, but it won't matter because the TTK of any gun is several times less than of any (even the best) melee weapon, resulting in you getting magdumped with near-perfect hit ratio and dying.

**Comment by EdenTheLiznerd** [comment]
> I honestly don't think this is necessary with the current state of melee balance. Nerfing armor penetration is a good change but the other nerfs are going a bit too far in the opposite direction. In the current state of combat melee weapons are not very effective even if the melee weapon in question has high kill potential (You won't get close enough to someone with a gun to actually hit them).
> 
> Additionally, botany being able to produce weaponizable plants adds alot of potential for in-round chaos and general silliness. Nerfing deathnettles to the point where they are basically unusable in combat is not the solution and seems like a bit of a knee-jerk nerf in response to a certain youtube video ;)

I... Have no clue what YouTube video you're talking about but ok, knee-jerk reaction it is I guess.

**Comment by deltanedas** [comment]
liltenhead ops

**Comment by Kadeo64** [comment]
sure, botanists CAN kill nukies, but if you're a competent nukie you will shoot them before they can get close. If you get killed by a deathnettle shitter it's your fault. I don't think deathnettles need a nerf. Also, nukies HAVE ways of healing from deathnettle, but they never bring them. You could easily whip up some ipepac or whatever to stop the amatoxin and then heal it with anti-toxin chems but nukie medics don't. also i'm like 80% sure ointment heals caustic anyway.

**Comment by deltanedas** [comment]
a pro shitter botanist 2 tapping a nukie after they had 3ms to react from jumping out of a locker, clearly the nukies fault

**Comment by Kadeo64** [comment]
>additionally, caustic is actually really annoying to heal, i think you only heal 2 per ointment which is a big cost considering it could be 5 burn instead

just change it to 5 burn then it's yml

**Comment by Kadeo64** [comment]
> a pro shitter botanist 2 tapping a nukie after they had 3ms to react from jumping out of a locker, clearly the nukies fault

maybe nukies should have checked their corners and brought shotguns

**Comment by Ilya246** [comment]
> > a pro shitter botanist 2 tapping a nukie after they had 3ms to react from jumping out of a locker, clearly the nukies fault
> 
> maybe nukies should have checked their corners and brought shotguns

- you're expecting nukies to be able to observe their entire screen at all times
- you're expecting nukies to be able to magically get rid of people camping behind doors waiting for nukies to open them
- you're expecting nukies to magically know what's inside of lockers, as a botanist could hide in a hallway locker and wait for nukies
- you're expecting nukies to have ultra gamer reaction time ang ping

this is basically 2-hit baton stunmeta but it actually kills you this time so teammates won't save you, and neither can an EMP implant

**Comment by Kadeo64** [comment]
> * you're expecting nukies to be able to observe their entire screen at all times

you have teammates, they should be observing as well

> * you're expecting nukies to be able to magically get rid of people camping behind doors waiting for nukies to open them

plant c4 on the door

**Comment by Ilya246** [comment]
> you have teammates, they should be observing as well

also people can just suddenly run at you from maint, where you can't see them, this happens rather often during nukies
> plant c4 on the door

and then the botanist hides behind the next door over until you run out of c4

**Comment by TrapFANchik** [comment]
It seems to me that this PR was made because the author / his friend was humiliated with the help of deathnettles. Nothing prevents you from using diloven. In general, I propose to remove all weapons at the stations except for standard tubes and tasers, because it can kill nuclear operatives, as well as remove omnisine from deus, because botanists have too strong a medicine, remove robust because botanists should not increase potency because it is too strong and easy

**Comment by Aisu9** [comment]
> a pro shitter botanist 2 tapping a nukie after they had 3ms to react from jumping out of a locker, clearly the nukies fault

everyone in this scenario have the ability to almost instantly kill the other still fair tbh

**Comment by TrapFANchik** [comment]
To be honest, it seems to me that the English/American community of SS14 is a bunch of whiners who are trying to get rid of everything that killed them at least once while playing the antagonist

**Comment by K-Dynamic** [comment]
> It seems to me that this PR was made because the author / his friend was humiliated with the help of deathnettles. Nothing prevents you from using diloven. In general, I propose to remove all weapons at the stations except for standard tubes and tasers, because it can kill nuclear operatives, as well as remove omnisine from deus, because botanists have too strong a medicine, remove robust because botanists should not increase potency because it is too strong and easy

Spite pr is still pr, let them cook

> you're just saying nukies being required to be ultra gamers to not die is good

Isn't that, uh... expected from nukies though?

Also anything can come from a locker, not just a botanist but a chem maxcap or secoff with batons

Lastly, spacing still exists and either kills botanist in locker or forces them in a slow-as-shit eva suit

**Comment by Ilya246** [comment]
> Also anything can come from a locker, not just a botanist but a chem maxcap or secoff with batons

yes chem bombs can be used that way and i think they're a bit good right now
and unlike a baton, a deathnettle is not countered by an EMP implant and actually kills you
> Lastly, spacing still exists and either kills botanist in locker or forces them in a slow-as-shit eva suit

lockers are airtight

**Comment by Aisu9** [comment]
> It seems to me that this PR was made because the author / his friend was humiliated with the help of deathnettles. Nothing prevents you from using diloven. In general, I propose to remove all weapons at the stations except for standard tubes and tasers, because it can kill nuclear operatives, as well as remove omnisine from deus, because botanists have too strong a medicine, remove robust because botanists should not increase potency because it is too strong and easy

now your just being petty and pathetic, and probably the most butthurt in this chain of "constructive" criticism.
really antagonising contributors? what is the next step doxing them?
2u per swing/projectile is if your not aware the standard standpoint for normal crew weaponry, Dylo heal extremely slowly and on top have a chance to make patient vomit(which slow them down heavily, and in this game speed is all)

**Comment by EmoGarbage404** [comment]
@TrapFANchik keep discussion civil and don't levy unfounded personal attacks onto PR authors.

For everyone else, keep discussion on topic.

**Comment by Aisu9** [comment]
i must say the normal nettle already doesn't pierce hardsuit, i could see the deathnettle only works on armoured target bo "balance things out" even a good deahtnettle will deal at most 33 damage per swing, and they swing one time per second

**Comment by K-Dynamic** [comment]
> yes chem bombs can be used that way and i think they're a bit too good right now and unlike a baton, a deathnettle is not countered by an EMP implant and actually kills you

> lockers are airtight

ty for confirming my long held suspicion, I will have fun with it :trollface: 

I'm of the opinion deathnettle are situational - they are extremely good melee weapons but guns have faster ttk (you better pray nukie has bad ping, is isolated or reloading), nukies can just sweep lockers and space areas to make melee users slower

I can see a problem when they're used for en-masse charges, but the botanist glove pr should resolve that

Either way it's near-certain death to use them since you're gonna be shot in the face or spaced

**Comment by TrapFANchik** [comment]
> @TrapFANchik keep discussion civil and don't levy unfounded personal attacks onto PR authors.
> 
> For everyone else, keep discussion on topic.

I sincerely apologize for not sticking my tongue deep into my ass, looking at how not only this PR is trying its best to kill hydroponics

**Comment by Aisu9** [comment]
> > yes chem bombs can be used that way and i think they're a bit too good right now and unlike a baton, a deathnettle is not countered by an EMP implant and actually kills you
> 
> I also agree chembombs suck but that'll be a different discussion for another day
> 
> > lockers are airtight
> 
> ty for confirming my long held suspicion, I will have fun with it :trollface:
> 
> I'm of the opinion deathnettle are situational - they are extremely good melee weapons but guns have faster ttk (you better pray nukie has bad ping, is isolated or reloading), nukies can just sweep lockers and space areas to make melee users slower
> 
> I can see a problem when they're used for en-masse charges, but the botanist glove pr should resolve that
> 
> Either way it's near-certain death to use them since you're gonna be shot in the face or spaced

then something must be done again spacing strategy, there is nothing the crew can defend themselves again spacing(no emergency help doesn't help, you will still die of cold) it's a long term fight not a random encounter against a mouse

**Comment by PTr1x** [comment]
> Deathnettles have had their ability to pierce armor be stripped and their damage values brought down alongside how much acid they can hold and how much it injects per hit
> tweak: Deathnettles are no longer able to bypass armor and don't do as much damage
it's not have any logic, because it's own feature of deathnettle. It's sound like "just delete deathnettle mutation".

**Comment by asperger-sind** [comment]
Edge-cases and overly complex theoreticals arguing with player expectations aside, this is an overnerf that makes deathnettles not worth the effort it takes to make them, that's not good and replaces one issue with another.

Having a 2-shot melee is pretty bad regardless of the scenario that its being used in, but having deathnettles be a downgrade from normal nettles is probably even worse. 

Expecting nukies to prepare and account for a dozen scenarios, half of them requiring setup before boarding, focusing entirely on the possibility of being attacked by a guy with a well-made deathnettle while also pursuing the original goal (disk) and dealing with other emergent situations not caused by the hypothetical deathnettle man is also absurd.

You shouldn't need to preemptively prepare and combat bullshit game mechanics in order to not have your round ruined when they make their move, an example of this is flashes and flashbangs to someone without flash resistance - you're slowed and blinded to such a degree where retaliation becomes impossible and flashes become a tool to instantly win fights, you should be disadvantaged because you lack the proper equipment to combat flashes, not completely defenseless praying for your opponent to have a massive skill issue.

My solution? Don't nerf deathnettles as hard, keep them an overall upgrade over normal nettles but make them slightly weaker so   the effects of bad game design don't roll over into bullshit game design territory.

**Comment by Kadeo64** [comment]
> My solution? Don't nerf deathnettles as hard, keep them an overall upgrade over normal nettles but make them slightly weaker so the effects of bad game design don't roll over into bullshit game design territory.

Just limit deathnettle chem storage to 10u and transfer to 3.33u then

**Comment by asperger-sind** [comment]
> then something must be done again spacing strategy, there is nothing the crew can defend themselves again spacing(no emergency help doesn't help, you will still die of cold) it's a long term fight not a random encounter against a mouse

Revert space pen removal, revive the grand wall of text of #24128, my foresight was great and predictions correct.

Alternatively buff emergency suits and re-introduce EVA capabilities for firesuits, spacing capabilities were already nerfed with the 2 PRs for structural damage rebalance and fireaxe getting trolled (twice) so the problem doesnt lie within spacing being too easy, the problem lies within spacing having a much greater effect on crewmembers without EVA (the majority of the station's crew) while also being an excellent area denial tool, I wrote a lot more about this in the space pen revert PR so look into that for more context and a more complete take on the matter.

**Comment by Jezithyr** [comment]
> > I honestly don't think this is necessary with the current state of melee balance. Nerfing armor penetration is a good change but the other nerfs are going a bit too far in the opposite direction. In the current state of combat melee weapons are not very effective even if the melee weapon in question has high kill potential (You won't get close enough to someone with a gun to actually hit them).
> > Additionally, botany being able to produce weaponizable plants adds alot of potential for in-round chaos and general silliness. Nerfing deathnettles to the point where they are basically unusable in combat is not the solution and seems like a bit of a knee-jerk nerf in response to a certain youtube video ;)
> 
> I... Have no clue what YouTube video you're talking about but ok, knee-jerk reaction it is I guess.

Fair enough, I was referring to the liltenhead video but if you made the PR for another reason that's fine. I'm not disagreeing that deathnettle needs a balancing pass, however this pr goes a bit too far in the other direction in terms of nerfing them.

**Comment by Emisse** [comment]
I think removing armor pen + lowering dmg is swinging too hard in the nerf direction for death nettles, they should still pose a significant threat at melee range and be one of the stronger melee options

**Comment by Jezithyr** [comment]
> I think removing armor pen + lowering dmg is swinging too hard in the nerf direction for death nettles, they should still pose a significant threat at melee range and be one of the stronger melee options

Good point, would just removing armor pen while slightly buffing acid damage be a better change? I feel like armor pen on something as simple as a plant is a bit silly imo.

**Comment by Emisse** [comment]
lets try it out

**Pull request merged**